cmd/cadvisor.go

```diff
@@ -106,6 +106,7 @@ var (
 		container.HugetlbUsageMetrics:            struct{}{},
 		container.ReferencedMemoryMetrics:        struct{}{},
 		container.CPUTopologyMetrics:             struct{}{},
+		container.ResctrlMetrics:                 struct{}{},
```

Comment:
please add this to the default ignoreMetrics above as well.

Reply:
Done.